examples/python/monitoring/08_custom_tools_monitoring.py

```
@@ -0,0 +1,315 @@
+
+Demonstrates performance monitoring for agents using custom tools:
+- Custom tool execution timing
+- Tool creation and registration performance
+- Tool usage pattern analysis
+- Tool efficiency optimization
+
+Shows how to monitor performance with custom-built tools.
+"""
+
+import time
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)